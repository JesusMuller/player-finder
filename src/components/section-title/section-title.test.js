import React from 'react';
import { shallow } from 'enzyme';
import SectionTile from './section-title';

describe('<SectionTitle />', () => {
    it('Should render an <h1> with a text inside', () => {
        const wrapper = shallow(<SectionTile textToShow="dummy title" />);
        expect(wrapper).toMatchSnapshot();
    });
});
