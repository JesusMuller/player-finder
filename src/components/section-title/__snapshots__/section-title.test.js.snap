// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render an <h1> with a text inside 1`] = `
<h1>
  dummy title
</h1>
`;
